bin/cqlsh.py

```diff
@@ -33,6 +33,7 @@
 import warnings
 import webbrowser
 from contextlib import contextmanager
+from cassandra.metadata import Murmur3Token
```

Comment:
@bschoening you need to move this lower around line 135, this is too soon to import anything from cassandra as it is not loaded yet.

Reply:
I was able to remove the import entirely by using self.conn.metadata.token_map.